### PR TITLE
Implemented ReceiveFile.

### PR DESCRIPTION
**Pull request by magnarn**
Added ExecuteScriptExecutor.
Handle script "storage: ReadLocalTextFile"

@torleifs Please review

### CODE REVIEW

File: Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    internal class ExecuteScriptExecutor : CommandExecutorBase
+    {
+        #region Methods
+
+        internal object ExecuteStorageScript(string command)
+        {
+            switch (command)
+            {
+                case "ReadLocalTextFile":
```
Comment (torleifs) [inline Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs]: These should probably be constants, but I guess that goes for all commands.

File: Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    #endregion
+
+    internal class ExecuteScriptExecutor : CommandExecutorBase
+    {
+        #region Methods
+
+        internal object ExecuteStorageScript(string command)
+        {
+            switch (command)
```
Comment (torleifs) [inline Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs:23]: Switch with only one case. Will it be extended in the future?

File: Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            {
+                case "ReadLocalTextFile":
+                    var args = ExecutedCommand.Parameters["args"] as JArray;
+                    if (args == null || args.Count == 0)
+                    {
```
Comment (torleifs) [inline Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs:28]: curly brace on line with condition

File: Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs (hunk 2 above)
Comment (torleifs) [inline Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs:23]: On second thought, Readability may be better with switch.

**Comment by torleifs** [comment]
Looks good!

File: Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs (hunk 3 above)
Comment (magnarn) [inline Winium.StoreApps.Driver/CommandExecutors/ExecuteScriptExecutor.cs:28]: I would agree, but that's not how it's done in the rest of the Winium project, so I'll stick to consistency.

**Pull request merged**